packages/gx-agent/src/agent/GXComplianceClient.ts

```diff
@@ -339,14 +349,13 @@ export class GXComplianceClient implements IAgentPlugin {
         verifiableCredentials: [verifiableCredentialResponse.verifiableCredential, args.complianceVC, args.selfDescriptionVC],
         challenge: args.challenge ? args.challenge : GXComplianceClient.getDateChallenge(),
         domain: participantDid,
-        persist: true,
+        persist: args.persist ? args.persist : false,
```

Comment:
Not sure why this is needed everywhere, simply pas in args.persist

Reply:
changed to just: `args.persist`